offload/test/offloading/gpupgo/pgo1.c

```diff
@@ -0,0 +1,82 @@
+// RUN: %libomptarget-compile-generic -fprofile-generate-gpu
```

Comment:
When targeting a V100, this command fails for me in both pgo1.c and pgo2.c.  In the LTO case:

```
LLVM ERROR: Circular dependency found in global variable set 
```

In the non-LTO case:

```
fatal error: error in backend: NVPTX aliasee must be a non-kernel function definition
```

I do not see this problem in PR #93365's pgo1.c.

Reply:
This is a limitation of the PTX target, globals cannot reference themselves. Most likely whatever NVIDIA engineer wrote the PTX parser found it annoying to reference something that wasn't fully parsed yet so he just decided to make it an error and here we are. See https://godbolt.org/z/53PP5c5ve.